release-please-config.json

```diff
@@ -0,0 +1,51 @@
+{
+    "packages": {
+      "catalog/bucket": {
+        "package-name": "bucket-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      },
+      "catalog/empty": {
+        "package-name": "empty-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      },
+      "catalog/gitops": {
+        "package-name": "gitops-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      },
+      "catalog/gke": {
+        "package-name": "gke-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      },
+      "catalog/hierarchy": {
+        "package-name": "hierarchy-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      },
+      "catalog/landing-zone": {
+        "package-name": "landing-zone-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      },
+      "catalog/log-export": {
+        "package-name": "log-export-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      },
+      "catalog/networking": {
+        "package-name": "networking-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      },
+      "catalog/project": {
+        "package-name": "project-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      },
+      "catalog/redis-bucket": {
+        "package-name": "redis-bucket-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      },
+      "catalog/spanner": {
+        "package-name": "spanner-blueprint",
+        "changelog-path": "CHANGELOG.md"
+      }
+    },
+    "bootstrap-sha": "TBD",
```

Comment:
Will be swapped in subsequent PR
https://github.com/googleapis/release-please/blob/master/docs/manifest-releaser.md#bootstrapping